kubetest2-gke/deployer/common.go

```diff
@@ -143,3 +164,49 @@ func buildProjectClustersLayout(projects, clusters []string, projectClustersLayo
 	}
 	return nil
 }
+
+func buildExtraNodePoolOptions(np string, enp *extraNodepool) error {
+	values, err := url.ParseQuery(np)
+	if err != nil {
+		return err
+	}
+	for k := range values {
+		v := values.Get("name")
+		switch k {
```

Comment:
I'd suggest a default case that errors so unrecognized params complain/error instead of getting ignored

Reply:
this is still outstanding